package.json

```diff
@@ -1,48 +1,50 @@
 {
-	"name": "css-chassis",
-	"title": "Chassis",
-	"description": "An attempt at creating open standards designed for CSS libraries, JavaScript UI libraries, and web developers in general",
-	"version": "0.0.1-pre",
-	"author": {
-		"name": "jQuery Foundation and other contributors",
-		"url": "https://github.com/jquery/css-chassis/blob/master/AUTHORS.txt"
-	},
-	"maintainers": [
-		{
-			"name": "Sarah Frisk",
-			"email": "sarah@sarahfrisk.net",
-			"url": "http://sarahfrisk.net"
-		},
-		{
-			"name": "Alexander Schmitz",
-			"email": "arschmitz@gmail.com",
-			"url": "http://arschmitz.me"
-		},
-		{
-			"name": "Chris Bracco",
-			"email": "chris@cbracco.me",
-			"url": "http://cbracco.me"
-		}
-
-	],
-	"repository": {
-		"type": "git",
-		"url": "git://github.com/jquery/css-chassis.git"
-	},
-	"bugs": "http://github.com/jquery/css-chassis/issues/",
-	"licenses": [
-		{
-			"type": "MIT",
-			"url": "https://github.com/jquery/css-chassis/blob/master/LICENSE.txt"
-		}
-	],
-	"dependencies": {},
-	"devDependencies": {
-		"commitplease": "2.0.0",
-		"grunt": "0.4.2",
-		"grunt-contrib-jshint": "0.10.0",
-		"grunt-git-authors": "2.0.0",
-		"grunt-jscs": "0.6.2"
-	},
-	"keywords": []
+  "name": "css-chassis",
+  "title": "Chassis",
+  "description": "An attempt at creating open standards designed for CSS libraries, JavaScript UI libraries, and web developers in general",
+  "version": "0.0.1-pre",
+  "author": {
+    "name": "jQuery Foundation and other contributors",
+    "url": "https://github.com/jquery/css-chassis/blob/master/AUTHORS.txt"
+  },
+  "maintainers": [
+    {
+      "name": "Sarah Frisk",
+      "email": "sarah@sarahfrisk.net",
+      "url": "http://sarahfrisk.net"
+    },
+    {
+      "name": "Alexander Schmitz",
+      "email": "arschmitz@gmail.com",
+      "url": "http://arschmitz.me"
+    },
+    {
+      "name": "Chris Bracco",
+      "email": "chris@cbracco.me",
+      "url": "http://cbracco.me"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/jquery/css-chassis.git"
+  },
+  "bugs": "http://github.com/jquery/css-chassis/issues/",
+  "licenses": [
+    {
+      "type": "MIT",
+      "url": "https://github.com/jquery/css-chassis/blob/master/LICENSE.txt"
+    }
+  ],
+  "dependencies": {},
+  "devDependencies": {
+    "commitplease": "2.0.0",
+    "grunt": "0.4.2",
+    "grunt-autoprefixer": "^2.1.0",
+    "grunt-contrib-jshint": "0.10.0",
+    "grunt-contrib-sass": "^0.8.1",
```

Comment:
We previously decided, to avoid an annoying ruby dependency when its likely most people have no other need for ruby, to use libsass. So this would be just `grunt-sass` for that.